jdbcSource/src/main/java/io/vantiq/extsrc/jdbcSource/JDBCCore.java

```diff
@@ -33,21 +34,22 @@
  */
 public class JDBCCore {
 
-    // vars for server configuration
     String sourceName;
     String authToken;
     String targetVantiqServer;    
     
     JDBCHandleConfiguration jdbcConfigHandler;
     
-    // vars for internal use
     Timer                       pollTimer = null;
     ExtensionWebSocketClient    client  = null;
     JDBC                        jdbc    = null;
     
-    // final vars
     final Logger log;
-    final static int    RECONNECT_INTERVAL = 5000;
+    final static int RECONNECT_INTERVAL = 5000;
+    final static int DEFAULT_BUNDLE_SIZE = 500;
```

Comment:
query limit (albeit not quite the same thing) is 1000 for VANTIQ queries when returning to an array.  That value got Mark into trouble, so might want to leave it as is -- just noting a consistency issue, sort of.